validator/client/src/test/java/tech/pegasys/teku/validator/client/BeaconProposerPreparerTest.java

```diff
@@ -158,20 +159,23 @@ void should_notCallPrepareBeaconProposerAfterFirstSlotOfEpoch() {
   @TestTemplate
   void should_callPrepareBeaconProposerAfterOnPossibleMissedEvents() {
     beaconProposerPreparer.onPossibleMissedEvents();
-    verify(validatorApiChannel).prepareBeaconProposer(any());
+    verify(validatorApiChannel).prepareBeaconProposer(any(), anyBoolean());
   }
 
+  @TestTemplate
```

Comment:
@rolfyone this was missing, I thought it was intended to be there.

Reply:
probably means the test wasn't running :/ yes it looks like it should follow the pattern here.